src/containers/Dashboard/DashboardContainer.tsx

```diff
@@ -13,28 +13,40 @@ import {useSelector, useDispatch} from "react-redux";
 import {RootState} from "../../app/rootReducer";
 import {useHistory} from "react-router-dom";
 import {Routes} from "../../constants/routes";
-import {getNodeList} from "./NodeSlice";
+import {getNodeList, getDiskInfo, getGeneralInfo} from "./NodeSlice";
 
 export const DashboardContainer = (): ReactElement => {
-    const [areElementsHidden, setElementsHidden] = useState(true);
+    const [areElementsHidden, setElementsHidden] = useState<boolean>(true);
+    const [selectedNodeIndex, setSelectedNodeIndex] = useState<number>(0);
     const history = useHistory();
-    const state = useSelector((state: RootState) => state);
     const dispatch = useDispatch();
+    const state = useSelector((state: RootState) => state);
+    const token = state.user.token;
+    const nodeList = state.node.nodeList;
 
     useEffect(() => {
-        if (!state.user.token) {
+        if (!token) {
             history.push(Routes.LOGIN_ROUTE);
         } else {
-            dispatch(getNodeList(state.user.token));
+            dispatch(getNodeList(token));
+            if (nodeList[0] && nodeList[0].id) {
+                dispatch(getGeneralInfo(token, nodeList[selectedNodeIndex].id));
+                dispatch(getDiskInfo(token, nodeList));
+                if (nodeList.length) setElementsHidden(false);
```

Comment:
Not sure what this condition means? It will be alway strue because you have above `if(noteList)[0]`